tests/Unit/Parallel/Test_TypeTraits.cpp

```diff
@@ -102,3 +119,139 @@ static_assert(Parallel::is_pupable_v<PupableClass>,
 static_assert(not Parallel::is_pupable<NonpupableClass>::value,
               "Failed testing type trait is_pupable");
 // [is_pupable_example]
+
+static_assert(
+    Parallel::is_chare_type<Parallel::Algorithms::Singleton, singleton>::value,
+    "Failed testing type trait is_chare_type for "
+    "Parallel::Algorithms::Singleton.");
```

Comment:
These "Failed testing X" messages are a relict from the past. You don't need them when the `static_assert` is descriptive enough by itself.